### ISSUE TITLE
[9.0]Pop error when I click Menu 'Inventory' if I use spansh language

### ISSUE THREAD

**Issue by duanyp1991** [odoo/odoo#10961]
**Pop error when I click Menu 'Inventory' if I use spansh language**

Impacted versions:

 - 9.0

Steps to reproduce:

 1. switch user language to Spanish
 2. click "inventory" menu

Current behavior:

 - Pop up error like this:
```
_Odoo Server Error
Traceback (most recent call last):
  File "/opt/odoo/sources/odoo/addons/report/controllers/main.py", line 101, in report_download
    response = self.report_routes(reportname, docids=docids, converter='pdf')
  File "/opt/odoo/sources/odoo/openerp/http.py", line 509, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/sources/odoo/addons/report/controllers/main.py", line 45, in report_routes
    pdf = report_obj.get_pdf(cr, uid, docids, reportname, data=data, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/addons/report/models/report.py", line 156, in get_pdf
    html = self.get_html(cr, uid, ids, report_name, data=data, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/addons/report/models/report.py", line 130, in get_html
    return self.render(cr, uid, [], report.report_name, docargs, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/addons/report/models/report.py", line 106, in render
    return view_obj.render(cr, uid, template, values, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/addons/website/models/ir_ui_view.py", line 142, in render
    return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/addons/web_editor/models/ir_ui_view.py", line 29, in render
    return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_ui_view.py", line 1070, in render
    return self.pool[engine].render(cr, uid, id_or_xml_id, qcontext, loader=loader, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/additional_addons/odoo_enterprise/website_version/models/ir_qweb.py", line 43, in render
    return super(QWeb, self).render(cr, uid, id_or_xml_id, qwebcontext, loader=loader, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_qweb.py", line 252, in render
    element = self.get_template(id_or_xml_id, qwebcontext)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_qweb.py", line 172, in get_template
    document = qwebcontext.loader(name)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_ui_view.py", line 1068, in loader
    return self.read_template(cr, uid, name, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_ui_view.py", line 943, in read_template
    return self._read_template(cr, uid, view_id, context=context)
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "<string>", line 2, in _read_template
  File "/opt/odoo/sources/odoo/openerp/tools/cache.py", line 85, in lookup
    value = d[key] = self.method(*args, **kwargs)
  File "/opt/odoo/additional_addons/odoo_enterprise/website_version/models/ir_ui_view.py", line 68, in _read_template
    arch = self.read_combined(cr, uid, view_id, fields=['arch'], context=context)['arch']
  File "/opt/odoo/sources/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/sources/odoo/openerp/addons/base/ir/ir_ui_view.py", line 640, in read_combined
    view_arch = etree.fromstring(view['arch'].encode('utf-8'))
  File "lxml.etree.pyx", line 3092, in lxml.etree.fromstring (src/lxml/lxml.etree.c:70473)
  File "parser.pxi", line 1828, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:106307)
  File "parser.pxi", line 1716, in lxml.etree._parseDoc (src/lxml/lxml.etree.c:105098)
  File "parser.pxi", line 1086, in lxml.etree._BaseParser._parseDoc (src/lxml/lxml.etree.c:99780)
  File "parser.pxi", line 580, in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:94254)
  File "parser.pxi", line 690, in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:95690)
  File "parser.pxi", line 620, in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:94757)
XMLSyntaxError: expected '>', line 42, column 70_
```

Expected behavior:

 - No error at all

**Comment by duanyp1991** [comment]
Hi @mart-e 

After our investigation, it's due to **a lot of** invalid translations of Spanish.eg, `</ strong>`, translators put a space in between.

**How to fix this issue:**
Identify all the translation items who has similiar mistake as above, correct them. We find the same mistakes in Chinese translation.

Every reviewer of each translation group should be aware of this before they accept translation terms.

**Question**:
Any script to automatic script to check this before we merge the translations from transfex?

Best regards,

**Comment by mart-e** [comment]
Hello,

What is the revision of your code? This should be fixed by now (both not crashing and correcting the errors in the translations)

**Comment by mart-e** [comment]
Should have been fixed at 6872aae8ef9c1a1fc6a8283aa9d12ce3a6859157 not to crash with wrong XML but most errors in Spanish should have been fixed in weekly synchronisation (did the few remaining this week).
If you can still reproduce with an up to date code, let me know
